### ISSUE TITLE
能否提供docker安装镜像

### ISSUE THREAD

**Issue by chenxudong2020** [ShadowsocksR-Live/shadowsocksr-native#228]
能否提供docker安装镜像，如题

**Comment by ssrlive** [comment]
不熟。

**Comment by Beginner-Go** [comment]
@ssrlive 能写一个openwrt平台的包吗
和 [这个](https://github.com/openwrt/packages/tree/master/net/shadowsocks-libev) 一样

**Comment by Echo-bash** [comment]
> 能否提供docker安装镜像，如题
可以参考我的
https://github.com/Echo-bash/Dockerfile/tree/master/shadowsocksr-n

**Comment by ssrlive** [comment]
感谢。能不能增加对 SSRoT 协议的支持？

**Comment by Echo-bash** [comment]
> 感谢。能不能增加对 SSRoT 协议的支持？因为目前 GFW 对 SS/SSR 协议基本上是秒封，没什么实际价值。

是修改over_tls_settings这个内容吗？

**Comment by ssrlive** [comment]
对。当然还得 购买域名，对这个域名设定dns解析指向你的docker主机的IP，安装 nginx，申请免费的tls证书， 最后依据前面的内容修改 `over_tls_settings`的内容。

但如果在 heroku 上，由于 证书 和 域名都有了，这些就免了，直接改 `over_tls_settings` 就好了。见 [这个项目](https://github.com/ShadowsocksR-Live/ssr2heroku) 。

**Comment by Echo-bash** [comment]
> 对。当然还得 购买域名，对这个域名设定dns解析指向你的docker主机的IP，安装 nginx，申请免费的tls证书， 最后依据前面的内容修改 `over_tls_settings`的内容。
> 
> 但如果在 heroku 上，由于 证书 和 域名都有了，这些就免了，直接改 `over_tls_settings` 就好了。见 [这个项目](https://github.com/ShadowsocksR-Live/ssr2heroku) 。
> 
> 另外，获取二进制文件时，使用这个链接，会自动获取最新版本的打包文件。不用理会版本的问题。 `https://github.com/ShadowsocksR-Live/shadowsocksr-native/releases/latest/download/ssr-native-linux-x64.zip`

嗯嗯docker 容器配置文件可以直接挂载的，前提是有https的网站代理SSRoT，使用SSRoT的话还可以免流就失效了吧，相当于真实访问nginx https网站了吧

**Comment by ssrlive** [comment]
对。所以客户端直接访问 nginx 的 443 端口。
